recipe/meta.yaml

```diff
@@ -148,4 +146,4 @@ extra:
   recipe-maintainers:
     - mxr-conda
     - h-vetinari
-    - oblute
+    - dsangillo
```

Comment:
You're still removing @oblute here, which - as I said in the previous comment - is not acceptable.

Reply:
Please remove yourself from the maintainers for the time being (see [this](https://github.com/conda-forge/gym-feedstock/pull/13#issuecomment-912820545) comment). You can contribute PRs to help move this feedstock forward without being a maintainer.

Reply:
This change request is still outstanding...